src/Hint/Extensions.hs

```diff
@@ -98,7 +98,7 @@ deriving instance Show Bar -- {-# LANGUAGE StandaloneDeriving #-}
 {-# LANGUAGE DeriveGeneric, GeneralizedNewtypeDeriving #-} \
 newtype Micro = Micro Int deriving Generic -- {-# LANGUAGE DeriveGeneric #-}
 {-# LANGUAGE DeriveGeneric, TypeFamilies #-} \
-data family Bar a; data instance Bar Foo = Foo deriving (Generic)
```

Comment:
Why did this need to change?

Reply:
gosh. i'm generally not in the habit of making random changes but in this instance i'm stumped - i have no idea why i did that and accordingly have reverted the change. sorry!

Reply:
good catch & thank you for the careful review!